caenbrew/packages/wxwidgets.py

```diff
@@ -0,0 +1,17 @@
+from ..packaging import AutotoolsPackage, package
+
+
+@package
+class WxWidgetsPackage(AutotoolsPackage):
+    """wxwidgets for GTK+"""
+
+    name = "wxwidgets"
+    homepage = "http://wxwidgets.org"
+    version = "3.0.2"
+
+    url = "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.2/wxWidgets-3.0.2.tar.bz2"  # noqa
+    artifacts = ["bin/wx-config", "include/wx", "lib/libwx*"]
```

Comment:
Is there actually a file called `lib/libwx*` created?

The artifacts are used only to determine if a package is installed. No globbing is done. You can just list one of the libraries that is installed if there are a lot (see https://github.com/arxanas/caenbrew/blob/master/caenbrew/packages/boost.py).